src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionCommentFilter.java

```diff
@@ -109,35 +109,52 @@
  * </li>
  * </ul>
  * <p>
- * To configure a filter to suppress audit events between a comment containing
- * {@code CHECKSTYLE:OFF} and a comment containing {@code CHECKSTYLE:ON}:
+ * To configure a filter to suppress audit events (MemberNameCheck,
+ * ConstantNameCheck and IllegalCatchCheck have been taken here for reference)
+ * between a comment containing {@code CHECKSTYLE:OFF} and a comment
+ * containing {@code CHECKSTYLE:ON}:
  * </p>
  * <pre>
- * &lt;module name="TreeWalker"&gt;
- *               ...
  *   &lt;module name="SuppressionCommentFilter"/&gt;
- *               ...
- * &lt;/module&gt;
- * </pre>
- * <p>
- * To configure a filter to suppress audit events between a comment containing line
- * {@code BEGIN GENERATED CODE} and a comment containing line {@code END GENERATED CODE}:
- * </p>
- * <pre>
- * &lt;module name="SuppressionCommentFilter"&gt;
- *   &lt;property name="offCommentFormat" value="BEGIN GENERATED CODE"/&gt;
- *   &lt;property name="onCommentFormat" value="END GENERATED CODE"/&gt;
- * &lt;/module&gt;
+ *   &lt;module name="MemberName"/&gt;
+ *   &lt;module name="ConstantName"/&gt;
+ *   &lt;module name="IllegalCatch"/&gt;
  * </pre>
  * <pre>
- * //BEGIN GENERATED CODE
- * &#64;Override
- * public boolean equals(Object obj) { ... } // No violation events will be reported
- *
- * &#64;Override
- * public int hashCode() { ... } // No violation events will be reported
- * //END GENERATED CODE
- * . . .
+ * class InputSuppressionCommentFilter
+ * {
+ *  int VAR1; // violation , Name 'VAR1' must match pattern '^[a-z][a-zA-Z0-9]*$'
+ *
+ *  //CHECKSTYLE:OFF
+ *  int VAR2; // suppressed violation
+ *  //CHECKSTYLE:ON
+ *
+ *  public static final int var3;
+ *  // violation above , Name 'var3' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *
+ *  //CHECKSTYLE:OFF
+ *  public static final int var4; // suppressed violation
+ *  //CHECKSTYLE:ON
+ *
+ *  public void method1() {
+ *
+ *      try{
```

Comment:
Fix indentation, it should be +2 for all

Reply:
Done.